src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -285,6 +410,53 @@ public static class Sse2
         /// </summary>
         public static Vector128<double> ConvertToDouble(Vector128<float> value) => ConvertToDouble(value);
 
+        
+        /// <summary>
+        /// double _mm_cvtsd_f64(__m128d a)
+        /// </summary>
+        public static double ConvertToDouble(Vector128<double> value) => ConvertToDouble(value);
+        /// <summary>
+        /// int _mm_cvtsd_si32 (__m128d a)
+        /// </summary>
+        public static int ConvertToInt32(Vector128<double> value) => ConvertToInt32(value);
+        /// <summary>
+        /// int _mm_cvtsi128_si32 (__m128i a)
+        /// </summary>
+        public static int ConvertToInt32(Vector128<int> value) => ConvertToInt32(value);
+        /// <summary>
+        /// __int64 _mm_cvtsd_si64 (__m128d a)
+        /// </summary>
+        public static long ConvertToInt64(Vector128<double> value) => ConvertToInt64(value);
```

Comment:
@fiigii, For instructions like this, which have an additional encoding on x64, how do we want to expose them?

Reply:
> an additional encoding on x64

Do you mean the 64-bit register in `cvtsd2si r64, xmm` ?
These intrinsics are only available in 64-bit mode, and [calling them in 32-bit should throw `PlatformNotSupportExeception`](https://github.com/dotnet/coreclr/blob/master/src/jit/hwintrinsicxarch.cpp#L296-L301).

```csharp
if (Sse2.IsSupported && Environment.Is64BitProcess)
{
    ulong res = Sse2.ConvertToInt64(vec);
}
```

Reply:
Alright, that sounds good to me.

I was mostly just wanting to confirm we were exposing them in `X86.Sse2` and not under some `X64` specific sub-class